.gitignore

```diff
@@ -1,3 +1,51 @@
+### JetBrains template
```

Comment:
This should be removed from the projects `.gitignore`. 

It should be part of your user specific `gitignore` (usually at `~/.gitignore` configured via `core.excludesfile`).